### DISCUSSION TITLE
ECONNREFUSED with cms-wordpress example starter

### DISCUSSION THREAD

**Discussion by San-Dan** [vercel/next.js#33437] [Help]
I just set up the cms-wordpress example, followed instructions precisely (except not doing the preview part) and the only code I've changed is to add my wp api url to .env.local, and that url looks like: 
`WORDPRESS_API_URL=https://localhost/*****/graphql`

Still, I get the error econnrefused when running npm run dev:
`FetchError: request to https://localhost/*****/graphql failed, reason: connect ECONNREFUSED 127.0.0.1:443`

I have gotten the same error when trying to set this up with another tutorial based on this starter, and no other versions have worked either, e.g. using an Apollo client.
Both projects are running locally - WP through XAMPP in Windows and Next in WSL (because it is super slow in Windows for me).

I had to install the WP GraphQL PLugin by searching in the plugin directory instead of using the zip-file. The zip could be installed but there was a fatal error when activating (I tried 2 different zips, latest and slightly older version).
`fatal error: uncaught typeerror: call_user_func_array(): argument #1 ($callback) must be a valid callback, function "graphql_activation_callback" not found or invalid function name`

My api url works fine in Postman, so does the url for just the rest api without graphql.

I'm trying out plain React and WP REST api tomorrow but I'm curious about the issue here, any ideas?

**Comment by joggienl** [comment]
This does not sound like a NextJS issue. NextJS does not use ssl (by default). Perhaps it is not able to connect from nonsecure to secure (is this happening in the browser or in the terminal).

If you are seeing this on the terminal, you might test to access the api with a simple GET via curl or wget, and maybe even with a simple nodejs script (just to test connectivity).

You could also try to connect to 127.0.0.1 directly, but that feels like shooting in the dark to me.
